OpenRA.Game/GameRules/WeaponInfo.cs

```diff
@@ -42,6 +42,8 @@ public sealed class WeaponInfo
 		[Desc("The sound played when the weapon is fired.")]
 		public readonly string[] Report = null;
 
+		public readonly bool OnlyPlayReportOnFirstBurst = false;
```

Comment:
Do we really, really need this? This is too much of a hack for me to like. Couldn't the Juggernaut use two weapons with the first weapon being the middle barrel voiced and the two other barrels having a silent Burst: 2 weapon instead?